hazelcast/src/main/java/com/hazelcast/jet/impl/connector/WriteJdbcP.java

```diff
@@ -117,6 +120,16 @@ public void init(@Nonnull Context context) {
                                                                exactlyOnce, batchLimit))
                                         .collect(Collectors.toList());
                     }
+
+                    @Override
+                    public Permission getRequiredPermission() {
+                        CommonDataSource commonDataSource = dataSourceSupplier.get();
+                        String jdbcUrl = null;
+                        if (commonDataSource instanceof DataSourceFromConnectionSupplier) {
+                            jdbcUrl = ((DataSourceFromConnectionSupplier) commonDataSource).getJdbcUrl();
+                        }
+                        return ConnectorPermission.jdbc(jdbcUrl, ACTION_WRITE);
```

Comment:
What about the case where user uses `dataSourceSupplier` instead of url. Do they need to `jdbc:*` as permission?

Reply:
yes